src/Recurr/Weekday.php

```diff
@@ -41,7 +41,7 @@ class Weekday
     /** @var int nth occurrence of the weekday */
     public $num;
 
-    protected $days = array('MO', 'TU', 'WE', 'TH', 'FR', 'SA', 'SU');
+    protected $days = array('SU', 'MO', 'TU', 'WE', 'TH', 'FR', 'SA');
```

Comment:
Is there an RFC that says what day is first ? (even if `$weekday` says 0 is Sunday)